assets/src/domain/shared/services/publicPath.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable 
+no-unused-vars,
+no-native-reassign,
+no-global-assign,
+@typescript-eslint/camelcase,
+@typescript-eslint/no-unused-vars,
+no-var
+*/
+/* global __webpack_public_path__ */
+declare var __webpack_public_path__: string;
+let assetsUrl = window?.eeEditorData?.assetsUrl;
+
+// TODO add assetUrl to global data for all domains
+// temporary workaround
+if (!assetsUrl) {
+	assetsUrl = window.location.origin + '/wp-content/plugins/event-espresso-core/assets/dist/';
```

Comment:
```
'/wp-content/plugins/event-espresso-core/assets/dist/'
```

we'll have to get this from the DOM because sometimes people rename the EE folder while FTPing files to their server

Reply:
Yes